server/lib/src/server/mod.rs

```diff
@@ -793,6 +794,14 @@ pub trait QueryServerTransaction<'a> {
             })
     }
 
+    fn get_domain_ldap_allow_unix_pw_bind(&mut self) -> Result<bool, OperationError> {
+        let res = self.internal_search_uuid(UUID_DOMAIN_INFO);
+        match res.unwrap().get_ava_single("domain_ldap_allow_unix_pw_bind"){
```

Comment:
needs to be the const, not a string

Reply:
you mean the attribute in `get_ava_single()`? Should I be passing the in const that represents the attrribute?

Reply:
You probably need to rebase, `get_ava_single` should only take an `Attribute`

Reply:
Makes sense, on rebasing, get_ava_single accpets Attributes